Apps/Src/Temperature.c

```diff
@@ -390,17 +389,15 @@ int32_t Temperature_GetTotalPackAvgTemperature(void){
  * @return SUCCESS or ERROR
  */
 ErrorStatus Temperature_SampleADC(uint8_t ADCMode) {
-#ifndef SIMULATION
-    //take control of mutex
-      RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
-    
-    LTC6811_adax(ADCMode, AUX_CH_GPIO1);							// Start ADC conversion on GPIO1
-    LTC6811_pollAdc();
```

Comment:
why is LTC6811_pollAdc() removed? Im actually kinda curious

Reply:
good catch